snap/snapcraft.yaml

```diff
@@ -0,0 +1,25 @@
+name: exa
+version: '0.7.0'
+summary: Replacement for 'ls' written in Rust
+description: |
+  It uses colours for information by default, helping you distinguish between
+  many types of files, such as whether you are the owner, or in the owning
+  group. It also has extra features not present in the original ls, such as
+  viewing the Git status for a directory, or recursing into directories with a
+  tree view. exa is written in Rust, so it’s small, fast, and portable.
+
+grade: stable
+confinement: strict
```

Comment:
I'm failing to build locally, so I can't test for myself, unfortunately. How are you installing the snap? strict snaps shouldn't have access to any files outside of the snap confinement (but if you've installed it in devmode, you wouldn't see this behaviour).

To confirm this, I created an `ls` snap and see this behaviour:

```
$ /snap/bin/ls
/snap/ls/x1//bin/ls: cannot open directory '.': Permission denied
$ /snap/bin/ls /
/snap/ls/x1//bin/ls: cannot open directory '.': Permission denied
```
but accessing its own files will work:

```
$ /snap/bin/ls ~/snap/ls
common	current  x1
$  /snap/bin/ls /snap/ls
current  x1
```

If you change confinement to "classic", then exa will be installed in a way that allows it full access to the system.